.coderabbit.yaml

```
@@ -0,0 +1,23 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+language: "en-CA"
+reviews:
+  path_instructions:
+    - path: "**/*.{cpp,h,hpp,java,js,jsx,tpp,ts,tsx}"
+      instructions: >-
+        - Prefer `false == <expression>` rather than `!<expression>`.
+  profile: "assertive"
+  poem: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)